pages/_checkout.jsx

```diff
@@ -21,12 +23,13 @@ function _CheckoutPage() {
         variables: getCheckoutPayload(items)
       });
 
-      window.location.assign(data.checkout.webUrl);
+      clearCart();
```

Comment:
In the Stripe starter we only clear the cart after a successful checkout. Is there some reason you're doing it here instead?

Reply:
Just realized I can add `shopify_checkout_success=true` to the url that redirects back to localhost:3000 after checkout is done. The fundamental problem is that shopify takes the lineitems and creates a checkout web URL, but there's no way to tell it to go back to your project without adding a redirect script into your store's checkout via the shopify admin. Manually adding my own query params to the URL that redirects back makes it so we know the checkout was successful.